functions.py

```diff
@@ -2,18 +2,16 @@
 import zipfile
 import boto3
 
-client = boto3.client('cloudformation')
-s3 = boto3.resource('s3')
-client_glue = boto3.client('glue')
+s3 = boto3.client('s3', region_name='ap-south-1')
```

Comment:
Naming of variables can be verbose enough to understand.
```suggestion
s3_client = boto3.client('s3', region_name='ap-south-1')
```